### PR TITLE
fixing on navbar

### PR DESCRIPTION
**Pull request by deveshksrivastava**
Migrate from Azure Pipelines to GitHub Actions:

- Removes the "old node" test, which just checked that installing on old
  node failed. We shouldn't need to test that
- Consolidates the build & test workflows so we don't end up with
  a proliferation of workflows.

### CODE REVIEW

File: src/Component/Pages/NavLink.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { NavLink } from 'react-router-dom';
+interface NavLinkProps {
+  href: string;
+  title: string;
```
Comment (DKSRIVASTAVA) [inline src/Component/Pages/NavLink.tsx:4]: please fix this.

File: src/valuable/body.tsx [hunk 2]
```diff
@@ -35,6 +35,10 @@ const Body = () => {
               <p>For Requesting AEE that enables you to enable AI</p>
             </div>
           </div>
+
+        </div>
+        <div>
+          <h2> For any Queries or feedback please contact <span> box.DAPmanagers@nationalgrideso.com </span></h2>
```
Comment (DKSRIVASTAVA) [inline src/valuable/body.tsx:41]: correct the spelling mistake

File: src/Component/Pages/NavLink.tsx (hunk 1 above)
Comment (deveshksrivastava) [reply src/Component/Pages/NavLink.tsx:4]: this has been fixed

File: src/valuable/body.tsx (hunk 2 above)
Comment (deveshksrivastava) [reply src/valuable/body.tsx:41]: done

**Review by DKSRIVASTAVA — APPROVED** [review]

**Pull request merged**